cluster-autoscaler/utils/backoff/backoff_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package backoff
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBackoffTwoKeys(t *testing.T) {
+	backoff := NewBackoff(time.Hour, 10*time.Minute, 3*time.Hour)
```

Comment:
nit: this (and every other test case in this file) effectively tests only max backoff, since it's always larger than init. Changing it here to max > init would improve coverage of that scenario, e.g.:
```NewBackoff(time.Hour, 10*time.Minute, 3*time.Hour)```

Reply:
We could also use a test case of incremental backoff scenario, sth like:
- backoff for init duration
- check that after this duration backoff is lifted
- backoff again
- check that this time backoff duration is 2x

Doesn't have to be in this PR, just a general note.